src/wbase/UserQueryInfo.h

```diff
@@ -32,6 +32,7 @@
 
 // Qserv headers
 #include "global/intTypes.h"
+#include "util/InstanceCount.h"
```

Comment:
Why this change here? I have inspected the header of `UserQueryInfo` and I'm not seeing any uses of `util::InstanceCount` here.

Reply:
Actually, I want to track `UserQueryInfo` as there aren't that many of them and they have a relatively complicated life cycle. Good catch.